src/components/Badge/Badge.tsx

```diff
@@ -28,14 +28,18 @@ export interface BadgeProps {
     | 'green-with-border'
   tooltip?: ReactNode
   isIndicator?: boolean
+  isTextUppercase?: boolean
+  weight?: '300' | '400' | '700'
 }
 
 export const Badge: FC<BadgeProps> = ({
   children,
   className = '',
   type = 'info',
   tooltip = null,
-  isIndicator = false
+  isIndicator = false,
+  isTextUppercase = true,
+  weight
```

Comment:
Powinienem tutaj dać weight = "400"?

Reply:
zostaw puste, to jest w tylu miejscach użyte, że sprawdzic każde to syzyfowa praca